experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -110,8 +114,33 @@ export const getAbsoluteUrl = (
   ) {
     host += `:${port}`;
   }
+  // Redact sensitive query parameters
+  if (path.includes('?')) {
+    const [pathname, query] = path.split('?', 2);
+    const searchParams = new URLSearchParams(query);
+    let sensitiveParamsToRedact: string[];
+
+    if (redactedQueryParams === undefined) {
```

Comment:
This guards against the case `null` is used as well

```suggestion
    if (redactedQueryParams == undefined) {
```